data/sean-scraper/scrape_state_cancer_profiles.py

```diff
@@ -1,261 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Just to verify: this is a vestigial file -- doesn't appear to be called even in the current `main` -- and you're now just downloading Sean's scraped releases directly, which he's presumably running regularly.